File: snippets/fsharp/System/String/IndexOf/simple1.fs
module simple1.fs
// <Snippet12>
open System

let str = "animal"
let toFind = "n"
let index = str.IndexOf "n"
printfn $"Found '{toFind}' in '{str}' at position {index}"

// The example displays the following output:
//        Found 'n' in 'animal' at position 1
// </Snippet12>

File: snippets/csharp/System/String/IndexOf/simple1.cs
using System;

public class IndexOfSimpleExample
{
    public static void Run()
    {
        // <Snippet12>
        string str = "animal";
        string toFind = "n";
        int index = str.IndexOf("n");
        Console.WriteLine($"Found '{toFind}' in '{str}' at position {index}");

        // The example displays the following output:
        //        Found 'n' in 'animal' at position 1
        // </Snippet12>
    }
}
